src/components/Document/write/IntegratedApplicationPreview.tsx

```
@@ -13,201 +15,63 @@ const IntegratedApplicationPreview = ({
-          {/* 검색한 위치를 보여주는 지도 */}
-          {renderMap(document?.address as GiggleAddress)}
-          <InputLayout title="Detailed Address" isEssential>
-            <div className="w-full self-stretch drop-shadow-[0_1px_2px_rgba(107,110,116,0.04)] rounded-xl flex items-center justify-start py-2.5 pr-3.5 pl-4">
-              <div className="w-full flex-1 relative">
-                {document?.address.address_detail}
+    <div className="w-full relative rounded-lg flex flex-col items center justify-center p-4 text-left body-3 bg-white">
+      <div className="w-full self-stretch flex flex-col items-start jusitfy-center">
+        <div className="w-full self-stretch flex flex-col items-center justify-start text-left pt-4 gap-3">
+          {Object.entries(document as IntegratedApplicationData).map(
+            ([key, value]) => (
+              <div className="w-full flex flex-col gap-1">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)